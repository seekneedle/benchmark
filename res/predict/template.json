{
    "content": "This is a good example.",
    "metadata": {
        "label": "negative"
    }
}