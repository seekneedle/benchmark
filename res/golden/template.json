{
    "content": "This is a good example.",
    "metadata": {
        "label": "positive"
    },
    "other": "hello world"
}